code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -12,6 +12,7 @@
 	var/obj/item/clothing/suit/space/hardsuit/suit
 	item_color = "engineering" //Determines used sprites: hardsuit[on]-[color] and hardsuit[on]-[color]2 (lying down sprite)
 	actions_types = list(/datum/action/item_action/toggle_helmet_light)
+	dynamic_hair_suffix = ""
```

Comment:
![3nb0tl3](https://user-images.githubusercontent.com/6209658/29643474-5aee8686-883d-11e7-98c7-df35b215c037.png)
![3nb0u3k](https://user-images.githubusercontent.com/6209658/29643478-5cc6ba8c-883d-11e7-9c3e-d7bc6e3cc5fa.png)

Looks fine to me?